### ISSUE TITLE
Support for nested imports

### ISSUE THREAD

**Issue by ikonst** [dropbox/mypy-protobuf#38]
What will it take to add support for nested imports? This seems to be supported in the Go version but not in Python.

**Comment by nipunn1313** [comment]
Hi! Thanks for the report!
Can you provide an example?

I'm out on vacation, but can hopefully take a look next week.

--Nipunn

**Comment by nipunn1313** [comment]
Hey @ikonst  - I added some testing around this in #39, but my test example actually passed just fine.
Is there a practical situation in which you're running into the issue? It's possible my test didn't cover the extent of the issue. Anyway - I'm hoping that makes it easier to proceed here.

**Comment by ikonst** [comment]
I managed to minimize it to:

- pb/my_message.proto:
  ```proto
  syntax = "proto3";
  package pb;
  message MyMessage {
      enum MyEnum {
          unknown = 0;
      }
  }
  ```

- pb/test.proto
  ```
  syntax = "proto3";
  package pb;
  import "pb/my_message.proto";
  message Test {
      pb.MyMessage.MyEnum my_enum = 1;
  }
  ```

**Comment by ikonst** [comment]
I'd be interested to take a shot at this, since our main protobufs at Lyft are failing with this error. It might take me some time to get to it, though.

**Comment by abhishekrb19** [comment]
@ikonst @nipunn1313 I'd be interested in this support too. Is this any progress or pointers on this? If not, I can take a shot at this.

**Comment by ikonst** [comment]
@abhishekrb19, I didn't get a chance to work on this yet.

**Comment by abhishekrb19** [comment]
@ikonst , I took a stab at it and was a relatively trivial change to make - please see https://github.com/dropbox/mypy-protobuf/pull/57

Let me know if you have any comments.

**Comment by abhishekrb19** [comment]
Just an update - this was supported in https://github.com/dropbox/mypy-protobuf/pull/57.
Do you want to mark this issue resolved?

**Comment by nipunn1313** [comment]
Yep thanks!

Fixed by #57
